src/applications/financial-status-report/config/form.js

```diff
@@ -60,6 +60,20 @@ const formConfig = {
           uiSchema: pages.veteranInfo.uiSchema,
           schema: pages.veteranInfo.schema,
           editModeOnReviewPage: true,
+          initialData: {
```

Comment:
Need to remove this or make null/undefined values and store in another file before merging